src/server.c

```diff
@@ -1081,8 +1106,12 @@ static void on_main_dispatch(void* aml_obj)
 		process_fb_update_requests(client);
 }
 
-EXPORT
-struct nvnc* nvnc_open(const char* address, uint16_t port)
+enum addrtype {
```

Comment:
definitions such as this one belong up top, between includes and function prototypes.